xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -1416,10 +1346,10 @@
         <Parameter Name="value" Type="System.Text.RegularExpressions.Group" RefType="out" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="key">A string with the group name key to look for.</param>
+        <param name="value">The out parameter where the retrieved group will be saved if found; <see langword="null" /> if not found.</param>
+        <summary>Attempts to retrieve a group identified by the provided name key, if it exists in the group collection.</summary>
+        <returns><see langword="true" /> if a group identified by the provided name key existed; <see langword="false" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if a group identified by the provided name key exists; <see langword="false" /> otherwise.</returns>
```